### PR TITLE
feat(SettingUp):creation of the project scaffoling

### LINKED ISSUE #1: Create a console app solution with a project to ServiceClient and a project to unit test of client

### PR DESCRIPTION
**Pull request by gsi-emiliogonzalez**
including libraries for dependency injection, console logger, spectre console, configuration

closes #1

### CODE REVIEW

File: DeribitSolution.sln [hunk 1]
```diff
@@ -0,0 +1,31 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 17
+VisualStudioVersion = 17.4.33122.133
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ConsoleApp", "DeribitSolution\ConsoleApp.csproj", "{032DB26F-3973-4EE9-8433-C20E2872E9D1}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ServiceClient", "ServiceClientLibrary\ServiceClient.csproj", "{CD9EBB22-06F5-468C-9625-0F45BCF17B84}"
+EndProject
```
Comment (gsi-yoancme) [inline DeribitSolution.sln:9]: remove Library from folder name

File: DeribitSolution/ConsoleApp.csproj [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="7.0.0" />
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="7.0.0" />
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="7.0.0" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Configuration" Version="7.0.0" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="7.0.0" />
+    <PackageReference Include="Spectre.Console" Version="0.46.0" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\ServiceClientLibrary\ServiceClient.csproj" />
```
Comment (gsi-yoancme) [inline DeribitSolution/ConsoleApp.csproj:21]: remove Library from the folder name

**Comment by gsi-robertogarcia** [comment]
I think this way is better because allow us to use the logger options from appsettings.json

```
var host = Host.CreateDefaultBuilder(args).Build();
var logger = host.Services.GetRequiredService<ILogger<Program>>();

logger.LogInformation("Host created.");

var qq = host.Services.GetService<IConfiguration>();

// reading from appsettings.json
var ewq = qq.GetSection("WebSocketSettings");
```

**Comment by gsi-robertogarcia** [comment]
appsettings.json

```
{
    "Logging": {
        "LogLevel": {
            "Default": "Information",
            "Microsoft": "Warning",
            "Microsoft.Hosting.Lifetime": "Information"
        }
    }
}
```

**Comment by gsi-emiliogonzalez** [comment]
> var host = Host.CreateDefaultBuilder(args).Build();
> var logger = host.Services.GetRequiredService<ILogger<Program>>();
> 
> logger.LogInformation("Host created.");
> 
> var qq = host.Services.GetService<IConfiguration>();
> 
> // reading from appsettings.json
> var ewq = qq.GetSection("WebSocketSettings");

Thanks, Robert, excellent observation. However, the intention of the code in Program.cs was to show that the libraries were added and everything was working fine. In this regard, the developer in charge of working in the Console could change the implementation to what you are proposing. 
Again this is just to prove that the scaffolding is working fine!

**Review by gsi-yoancme — APPROVED** [review]

**Comment by gsi-robertogarcia** [comment]
I think this should be the initial code in program.cs

```
using Microsoft.Extensions.Configuration;
using Microsoft.Extensions.DependencyInjection;
using Microsoft.Extensions.Hosting;
using Microsoft.Extensions.Logging;

var host = Host
    .CreateDefaultBuilder(args)
    .ConfigureServices(services =>
        {
            services.AddSingleton<IXxx, Xxx>();
        })
    .Build();

var logger = host.Services.GetRequiredService<ILogger<Program>>();
var configuration = host.Services.GetService<IConfiguration>();
```

**Pull request merged**